Standards/CCSMB-10.md

```diff
@@ -0,0 +1,14 @@
+# *CCSMB 10:* Specifying where programs and libraries are installed and where user data is stored
+
+*Author: @Rainb0wSkeppy*  
+*Version: vX.Y.Z*  
+*Last updated: YYYY-MM-DD*  
+
+## `settings` API settings
+Compliant installers MUST install programs into the directory specified through the `path.programs` setting and MUST install libraries into the directory specified through the `path.libraries` setting.
+The installers SHOULD NOT run if the settings aren't set.  
```

Comment:
It feels a bit weird that the installer should exit if the settings are not present. I would instead maybe print a log message saying it supports CCSMB-10?

Reply:
Maybe it should ask where to put programs and then set the setting

Reply:
i removed that part